clouddrift/dataformat.py

```diff
@@ -60,20 +64,20 @@ def from_files(
         cls,
         indices: list,
         preprocess_func: Callable[[int], xr.Dataset],
-        vars_coords: dict,
-        vars_meta: list = [],
-        vars_data: list = [],
+        vars_coords: list,
+        vars_meta: Optional[list] = [],
+        vars_data: Optional[list] = [],
```

Comment:
`vars_` doesn't seem to me like it adds meaning or distinction. What do you think about just `coords`, `meta`, and `data`?

```suggestion
        coords: list,
        meta: Optional[list] = [],
        data: Optional[list] = [],
```